### DISCUSSION TITLE
Will Lowdefy support Cloudflare Pages Functions?

### DISCUSSION THREAD

**Discussion by loonfly** [lowdefy/lowdefy#1115] [Questions and Help] [answered]
I love how Lowdefy works!

Will Lowdefy support deployment to cloudflare pages?

cloudflare pages has the `functions` just like netlify functions.

I know this ain't easy things, just curious.  I am new to Lowdefy anyway 💖

**Comment by SamTolmay** [comment, marked as answer]
Hi @loonfly 

I am glad you are enjoying using Lowdefy 😃.

We are planning on adding a feature that will allow you to build static apps, that don't require a server. These can be hosted on Cloudflare pages, but they do come with some limitations, namely no support for user authentication or any connections/requests.
You can look a these for more details: #639 #537 

We would really like to support Cloudflare pages a workers, and running a full Lowdefy server on the CLoudflare workers serverless platform is something we have talked about. We do expect it to be challenging to do though. Cloudflare workers don't run Node.js, but rather a custom V8 runtime, without things like file access. We also expect some connection libraries won't work in this runtime.

**Reply by loonfly** [reply]
Ah nice!. thanks. that's clear explanation. btw `feature that will allow you to build static apps` is good enough and will add lot more use cases!
